### PR TITLE
ci: new identity

### PR DESCRIPTION
**Pull request by stepansergeevitch**
Update integration testing workflow to use a new setup action v2, which creates new identity engines and databases

### CODE REVIEW

**Review by alexradzin — APPROVED** [review]

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -8,12 +8,12 @@ on:
         required: false
         default: ''
       engine:
-        description: 'Engine'
-        required: true
+        description: 'Engine - a new one will be created if not provided'
+        required: false
```
Comment (ptiurin) [inline .github/workflows/integration-test.yml:12]: This is  somewhat misleading, if database is provided but engine is blank we'll break on `find-engine-name` step. Should we check that both inputs are either defined or undefined?

Comment (stepansergeevitch) [reply .github/workflows/integration-test.yml:12]: Makes sense, will add this

File: .github/workflows/integration-test.yml [hunk 2]
```diff
@@ -42,25 +42,29 @@ jobs:
...
+             echo "SERVICE_ACCOUNT_ID=${{ secrets.FIREBOLT_CLIENT_ID_NEW_IDN }}" >> "$GITHUB_ENV"
+             echo "SERVICE_ACCOUNT_SECRET=${{ secrets.FIREBOLT_CLIENT_SECRET_NEW_IDN }}" >> "$GITHUB_ENV"
           fi
+
+      - name: Validate database and engine
+        if: ${{ (github.event.inputs.database == '') != (github.event.inputs.engine == '') }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+            core.setFailed("Database and Engine parameters should be provided simultaneously")
```
Comment (ptiurin) [inline .github/workflows/integration-test.yml:57]: Nitpicking, but let's set this as a first step in a workflow. No point in checking out the code and setting up java if we were to fail anyway. Saves some time.

**Review by ptiurin — APPROVED** [review]
Apart from that lgtm.

**Pull request merged**